transformer.py

```diff
@@ -378,7 +378,7 @@ def _get_activation_fn(activation):
 
 
 def build_transformer(configuration):
-    return Transformer(d_model=6, nhead=6, dropout=configuration.dropout)
+    return Transformer(d_model=32, nhead=8, dropout=configuration.dropout)
```

Comment:
How about converting d_model (better called "embedding_dimension") and number of heads to variables in the config file? Also, I think it's better to delete this function completely, as it will be clearer to simply use the Transformer class directly to create an instance of it anywhere it's needed.